packages/circle-react-components/.nvmrc

```diff
@@ -0,0 +1 @@
+v16.18.1
```

Comment:
There are other parts of this project (`MAINTAINERS_NOTES` and `package.json`) that state that this project is using Node v20 or higher.